mintlify/mintlify
# Peter private google docs integrations

Implement Google Docs installation.
### How to test it 
1. Delete the Google integration from the database.
2. Click on the Google icon under the `/automations` page 
![Screen Shot 2022-06-09 at 4 32 37 PM](https://user-images.githubusercontent.com/40123613/172961907-7b8b54c0-65e4-406d-bde5-4da64cb948a2.png).
3. Select your Google account on the consent screen, and hit accept.
4. See the Google Docs show up as installed on the `/settings/organization#integrations` page.
![Screen Shot 2022-06-09 at 4 33 46 PM](https://user-images.githubusercontent.com/40123613/172961976-9cccc72f-e559-4d35-bcd1-ef57b1ea5d56.png)
